ports/ghostris/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Ghostris.sh</path>
+    <name>Ghostris</name>
+    <desc> Ghostris is a puzzle game where the blocks you place down are haunted and will do stuff behind your back as you try to clear lines. You need to think not just about how to match up blocks, but also how to deal with the blocks' movement patterns, and react to ghosts showing up to interfere with your puzzling. Can you survive the Ghostris?</desc>
+    <releasedate>20191129T000000</releasedate>
+    <developer>Yal</developer>
+    <publisher>Yal</publisher>
+    <genre>Puzzle</genre>
+    <image>./ghostris/screenshot.png</image>
```

Comment:
![ghostrist-fs8](https://github.com/user-attachments/assets/16844737-671f-4ed6-9339-6ca63071422d)
:)

Reply:
Thanks, I will add the cover :)